nestkernel/model_manager.cpp

```diff
@@ -547,48 +543,23 @@ ModelManager::memory_info() const
   std::cout.unsetf( std::ios::left );
 }
 
-void
-ModelManager::create_secondary_events_prototypes()
-{
-  delete_secondary_events_prototypes();
-  secondary_events_prototypes_.resize( kernel().vp_manager.get_num_threads() );
-
-  for ( thread tid = 0; tid < static_cast< thread >( kernel().vp_manager.get_num_threads() ); ++tid )
-  {
-    secondary_events_prototypes_[ tid ].clear();
-    for ( synindex syn_id = 0; syn_id < connection_models_[ tid ].size(); ++syn_id )
-    {
-      if ( not connection_models_[ tid ][ syn_id ]->is_primary() )
-      {
-        secondary_events_prototypes_[ tid ].insert(
-          std::pair< synindex, SecondaryEvent* >( syn_id, connection_models_[ tid ][ syn_id ]->create_event() ) );
-      }
-    }
-  }
-}
-
 synindex
 ModelManager::register_connection_model_( ConnectorModel* cf )
 {
   if ( synapsedict_->known( cf->get_name() ) )
   {
     delete cf;
-    std::string msg = String::compose(
-      "A synapse type called '%1' already exists.\n"
-      "Please choose a different name!",
-      cf->get_name() );
+    std::string msg =
+      String::compose( "A synapse type called '%1' already exists.\nPlease choose a different name!", cf->get_name() );
```

Comment:
You use here `cf` which you delete one line above. Switch order (Github doesn't let me make a suggestion here, unfortunately).

Reply:
Done.